scenarioo-client/test/e2e/specs/browse_detailarea.ts

```diff
@@ -0,0 +1,96 @@
+                await DetailAreaPage.assertDetailsAreaExpanded();
+                await DetailAreaPage.assertSectionExpanded('Build');
+                await DetailAreaPage.assertSectionExpanded('Branch');
+                await step('Detailarea and important sections are expanded on first visit');
+
+                await DetailAreaPage.collapseDetailsSection('Build');
+                await DetailAreaPage.assertSectionCollapsed('Build');
+                await step('Build section has been collapsed');
+
+                await DetailAreaPage.collapseDetailsArea();
+                await DetailAreaPage.assertDetailsAreaCollapsed();
+                await step('Detailarea has been collapsed');
+
+                await Utils.refreshBrowser();
+                await DetailAreaPage.assertDetailsAreaCollapsed();
+                await step('Collapsed detailarea is remembered on revisit.');
+
+                await DetailAreaPage.expandDetailsArea();
+                await DetailAreaPage.assertDetailsAreaExpanded();
+                await step('Detailarea has been expanded');
+
+                await DetailAreaPage.assertSectionExpanded('Branch');
+                await DetailAreaPage.assertSectionCollapsed('Build');
+                await step('Collapsed or expanded areas are remembered on revisit');
+
+                await DetailAreaPage.expandDetailsSection('Build');
+                await DetailAreaPage.assertSectionExpanded('Build');
+                await step('Build section has been expanded');
+
+                await DetailAreaPage.collapseDetailsSection('Branch');
+                await DetailAreaPage.assertSectionCollapsed('Branch');
+                await step('Branch section has been collapsed');
+
+                await Utils.refreshBrowser();
+                await DetailAreaPage.assertSectionCollapsed('Branch');
+                await DetailAreaPage.assertSectionExpanded('Build');
+                await step('Collapsed or expanded areas are remembered on revisit');
+            });
+
+        scenario('Remembers collapsed state of details area and sections on scenarios overview')
+            .description('States of detailarea and sections are correctly saved in LocalStorage.')
+            .it(async () => {
+                await HomePage.selectUseCase(SECOND_USE_CASE);
+                await step('select a scenario in the scenario list');
+
+                await DetailAreaPage.assertDetailsAreaExpanded();
+                await DetailAreaPage.assertSectionExpanded('Use Case');
+                await DetailAreaPage.assertSectionCollapsed('Labels');
+                await DetailAreaPage.assertSectionCollapsed('Webtest Class');
+                await step('Detailarea and important sections are expanded on first visit');
+
+                await DetailAreaPage.collapseDetailsSection('Use Case');
+                await DetailAreaPage.assertSectionCollapsed('Use Case');
+                await step('Build section has been collapsed');
```

Comment:
copy paste comment "Build"

Reply:
would be cool to expand another one, which is collapsed by default and then chekc both on revisit